classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :student
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(student)
+    @student.push(student)
+    student.classroom = self
+  end
```

Comment:
- [ ] Great job for adding the add student to the classroom but also remeber when you are associate relationships so also you should add define a method add_classroom to the student but I can't see any update in `student.rb` file so please add the add_classroom method to your student.rb file as well 🚀🔥🚀✨✅

- this is a method that serves as the 'belongs-to' side (a student belongs to a classroom) you can check this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more information. 🚀✨